src/colorizer/Dataset.ts

```diff
@@ -29,6 +29,9 @@ export type DatasetMetadata = {
     height: number;
     units: string;
   };
+  frameDurationSeconds: number;
+  /* Optional offset for the timestamp. */
+  startTimeSeconds: number;
```

Comment:
Somewhat debatable how useful this is, but I figured it could be useful if we ever decide to show a slice of a dataset (or want to set this in terms of like, say, colony time.)

Reply:
YAGNI 😉 but no need to remove it now.